src/content/learn/add-react-to-an-existing-project.md

```diff
@@ -1,59 +1,59 @@
 ---
-title: Add React to an Existing Project
+title: Přidání Reactu do existujícího projektu
 ---
 
 <Intro>
 
-If you want to add some interactivity to your existing project, you don't have to rewrite it in React. Add React to your existing stack, and render interactive React components anywhere.
+Pokud máte v plánu přidat interaktivitu do svého existujícího projektu, není nutné celý projekt přepracovat do Reactu. Můžete jednoduše začlenit React mezi již použité technologie a umístit interaktivní React komponenty, kamkoliv je potřeba.
```

Comment:
I see great improvement there! Maybe the "do svého existujícího projektu" sounds a bit odd. What about "do existujícího projektu"?